Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -45,9 +45,9 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
            export NAME="$(whoami)-$RANDOM"
            # Create the node pool with the following taint to guarantee that
-           # Pods are only scheduled in the node when Cilium is ready.
+           # Pods are only scheduled/executed in the node when Cilium is ready.
            gcloud container clusters create "${NAME}" \
-            --node-taints node.cilium.io/agent-not-ready=true:NoSchedule \
+            --node-taints node.cilium.io/agent-not-ready=true:NoExecute \
```

Comment:
based on my understanding, NoExecute and NoShedule is not overlapping, do you think we need to have both ? 

- NoSchedule is to make sure no new pod created util cilium is ready
- NoExecute is to make sure that existing pods get evicted.

Reply:
That's a great question — @aanm asked the same. Based on the [documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), I believe they're indeed overlapping:

> if there is at least one un-ignored taint with effect `NoExecute` then the pod will be evicted from the node (if it is already running on the node), *and* will not be scheduled onto the node (if it is not yet running on the node).

So I think we're good with `NoExecute`.

Reply:
thanks, I might have missed such convo :100: